packages/lesmis/src/views/DashboardView.js

```diff
@@ -439,23 +479,30 @@ export const DashboardView = React.memo(() => {
           <VitalsView vitals={vitals} />
         </Container>
       </Wrapper>
-      {dropdownOptions.map(({ value, Body, Component }) => (
+      {dropdownOptions.map(({ value, Body, Component, useYearSelector, ComponentProps }) => (
         <TabPanel key={value} isSelected={value === selectedOption} Panel={React.Fragment}>
           <Component
             entityCode={entityCode}
+            year={useYearSelector && selectedYear}
             Body={Body}
-            TabSelector={
-              <StyledSelect
-                id="dashboardtab"
-                options={dropdownOptions}
-                value={selectedOption}
-                onChange={handleChange}
-                showPlaceholder={false}
-                SelectProps={{
-                  MenuProps: { disablePortal: true },
-                }}
-              />
-            }
+            TabBarLeftSection={() => (
+              <TabBarSection>
+                <StyledSelect
+                  id="dashboardtab"
+                  options={dropdownOptions}
+                  value={selectedOption}
+                  onChange={handleChange}
+                  showPlaceholder={false}
+                  SelectProps={{
+                    MenuProps: { disablePortal: true },
+                  }}
+                />
+                {useYearSelector ? (
+                  <YearSelector value={selectedYear} onChange={setSelectedYear} />
+                ) : null}
```

Comment:
```suggestion
              {useYearSelector && (
                  <YearSelector value={selectedYear} onChange={setSelectedYear} />
                )}
```

Reply:
This is surprising but handy! I didn't realise they had special handling to treat `false` unlike other values during JSX rendering 
![image](https://user-images.githubusercontent.com/1274422/123336480-06c22a00-d59a-11eb-88cb-7e3c717a93e6.png)

Reply:
https://reactjs.org/docs/conditional-rendering.html#inline-if-with-logical--operator

Reply:
Yeah this bit is what had confused me from that doc
![image](https://user-images.githubusercontent.com/1274422/123337322-26a61d80-d59b-11eb-9cc6-e3554432fd76.png)